data/json/items/vehicle/engineering.json

```diff
@@ -47,8 +47,8 @@
     "symbol": "&",
     "name": { "str": "rockwheel" },
     "color": "dark_gray",
-    "description": "A large and heavy jagged metal disc to dig trenches.",
-    "weight": "595 kg",
+    "description": "A large, heavy, jagged metal disc for digging trenches.",
+    "weight": "62000 g",
```

Comment:
Vehicle drag values changed.  Update values in `data/mods/TEST_DATA/vehicle_drag_test.json` with values printed below:
`{ "trencher": [ 0.520838, 1.386882, 1576.078846, 8506, 8828 ] },`

Reply:
should be all set.